bip-emilengler-redefinition-of-the-term-address.mediawiki

```diff
@@ -0,0 +1,54 @@
+<pre>
+  BIP: UNDEFINED
+  Titel: Redefinition of the term address
+  Author: Emil Engler <me@emilengler.com>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: Public Domain
```

Comment:
BIP2 recommends to use CC0 instead. And I think all new BIPs did that.

Reply:
Understandable, some countries (e.g Germany) don't have the US like public domain